src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
+
+            for key, value in validate_image_labels(image_row.labels).items():
+                if key[:11] == "ai.backend.":
```

Comment:
How about just printing the full name of key?
Is it requested to trim out `ai.backend.` when print labels?

Reply:
![스크린샷 2024-02-14 오후 3 24 20](https://github.com/lablup/backend.ai/assets/91869302/2f157aa6-ec8e-40d8-a715-bfc0c63d137e)
I didn't receive any requests, but I tried to trim unnecessary parts in order to look neat when printing.